helper-cli/src/main/kotlin/commands/ExportPathExcludesCommand.kt

```diff
@@ -75,7 +75,11 @@ internal class ExportPathExcludesCommand : CommandWithHelp() {
             .replaceConfig(repositoryConfigurationFile.readValue())
             .getRepositoryPathExcludes()
 
-        val globalPathExcludes = pathExcludesFile.readValue<RepositoryPathExcludes>()
+        val globalPathExcludes = if (pathExcludesFile.isFile) {
```

Comment:
"target file" in the commit message sounds like a file to write to. Better say "Do not require a global path excludes file to exist".

Reply:
Well, the `global path excludes file` actually is being written to and for that reason I called it `target file`. Done anyway - to me this change request feels quite unimportant, do you feel the same? ..and if yes can we make such change requests optional in the future and mark them as such to speed-up the review process a bit ?

Reply:
Actually, to me it just sounded wrong, and I though you might have confused the message with the other commit about the "target dir" and thus were accidentally using the word "target" here, too. So, thanks for fixing this up.